cli/rm.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// RmCommand is used to implement 'rm' command.
+type RmCommand struct {
+	baseCommand
+	force bool
+}
+
+// Init initializes RmCommand command.
+func (r *RmCommand) Init(c *Cli) {
+	r.cli = c
+	r.cmd = &cobra.Command{
+		Use:   "rm container",
+		Short: "Remove one or more containers",
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
Add `Long` and `Example` in command definition like https://github.com/alibaba/pouch/blob/master/cli/create.go#L28-L33 ?
Then we can automatically generate docs?  How about that?